client/packages/system/src/Patient/PatientView/PatientView.tsx

```diff
@@ -206,13 +208,17 @@ export const PatientView = () => {
           }}
         />
       ) : null}
-      <AppBarButtons disabled={!!createNewPatient} />
+      <AppBarButtons disabled={!!createNewPatient} store={store} />
       <PatientSummary />
-      {/* Only show tabs for saved patients */}
+      {/* Only show tabs if program module is on and patient is saved.
```

Comment:
I was going to complain about the multiple `?` operators and refactored to the following, then realised that this is just what the comment says it should be..

```
      {!createNewPatient && store?.preferences.omProgramModule ? (
        <DetailTabs tabs={tabs} requiresConfirmation={requiresConfirmation} />
      ) : (
        <PatientDetailView onEdit={setIsDirtyPatient} />
      )}
```

Reply:
I just re-read it 😅 
[ad327d8](https://github.com/openmsupply/open-msupply/pull/1998/commits/ad327d846bb2d0d7e95a94d30e8df45fadcdb5fe)